include/dev/INA138.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef LVSS_INA138_HPP
+#define LVSS_INA138_HPP
+
+#include <EVT/io/ADC.hpp>
+#include <EVT/utils/log.hpp>
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
+namespace log = EVT::core::log;
+
+namespace LVSS {
+
+/**
+ * Class for LVSS current sensor
+ */
+class INA138 {
+public:
+    /**
+      * Constructor for current sensing class
```

Comment:
This is the smallest of nitpicks, but your comment blocks are off by one space
```suggestion
     * Constructor for current sensing class
```